dynamics-nav/SETRECFILTER-Function--RecordRef-.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "SETRECFILTER Function (RecordRef)"
+description: SETRECFILTER Function (RecordRef) Sets a filter on a record that is referred to by a RecordRef.
```

Comment:
Too short per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.